public/javascripts/labelingGuidePanelResize.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Call this function when the user re-sizes the window. Then, the first time the window width becomes too small
+ * to hold the main content (<1185px), re-size the help panel to 250px instead of 275px. The second time the
+ * width becomes too small (<978px), change the panel to just be static, full-width, and get rid of the
+ * scrolling for the panel if it's there. When the window again becomes >=978px, change the panel back.
+ */
+function updateSidebarForWindowSize() {
+    var w = document.documentElement.clientWidth;
+    var smallWindowWidth = 978;
```

Comment:
Okay sorry for just realizing this now... But I think the better way to design this would be to use CSS classes. These functions should check for when the page width becomes large/small enough, and then it should add/remove CSS classes that hold all this style info. I would guess there would be a separate CSS class for when it is a sidebar vs when it is not. And when the width becomes to small, the sidebar class is removed from the div and the full width one is added. I would guess that you could use jquery for this.